helpers/Matrix.h

```diff
@@ -0,0 +1,39 @@
+#ifndef _HELPERS_MATRIX_H_
+#define _HELPERS_MATRIX_H_
+
+#include "include/z64.h"
+#include "symbols.h"
+
+#define Matrix_RotateY_s(binang, A) Matrix_RotateY(BINANG_TO_RAD(binang), A)
+#define Matrix_RotateX_s(binang, A) Matrix_RotateX(BINANG_TO_RAD(binang), A)
+#define Matrix_RotateZ_s(binang, A) Matrix_RotateZ(BINANG_TO_RAD(binang), A)
+
+#define Matrix_RotateY_f(degf, A)   Matrix_RotateY(DEGF_TO_RADF(degf), A)
+#define Matrix_RotateX_f(degf, A)   Matrix_RotateX(DEGF_TO_RADF(degf), A)
+#define Matrix_RotateZ_f(degf, A)   Matrix_RotateZ(DEGF_TO_RADF(degf), A)
+
+void Matrix_MultX(f32 scale, Vec3f* dst) {
```

Comment:
Considering this is a header these functions should all be static, and possibly inline

Reply:
What difference does that make?

Reply:
The difference is less relevant here but if you tried to using these functions anywhere in decomp got example it would not compile. I guess the intention though is that if you were using decomp that you'd add these to sys_matrix..?

Reply:
In that case it should be fine as it is? I couldn't figure a situation where someone would use z64hdr for decomp. And if someone wants to use these in decomp, then it would be copy paste job to somewhere in decomp, like sys_matrix as you mentioned.

Reply:
Oh actually i thought you were replying to my other comment, these need to be static in case you link multiple c files together so you dont have a definition conflict if both are including this header

Reply:
Another thing is that if these aren't static they will always be included in your file, regardless of if they are even used, which increases code size.

Reply:
That's why it has #ifndef guard. And this is not going to be included by default. User includes it only if needed. Also I've had static inline functions written into actor even if I did not use them.

Reply:
That only solves the issue of including it in the same file twice, if you include it in two files and link them together then it will have a definition error